src/Controllers/SchemaController.js

```diff
@@ -621,11 +630,26 @@ class SchemaController {
         found = true;
       }
     }
-    if (!found) {
-      // TODO: Verify correct error code
-      throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND,
+     
+    if (found) {
+      return Promise.resolve();
```

Comment:
the logic is that Pointer Perms is inferred later on the query itself, so if the general rule is granted for a user, we can pass as this will be enforced later?